site/content/docs/cicd/silent.md

```diff
@@ -0,0 +1,11 @@
+#Deploy with —silent
+
+You can use AWS Deploy Tool when developing your app using any Continuous Deployment system. Continuous Deployment systems let you automatically build, test and deploy your application each time your check in updates to your source code. Before you can use AWS Deploy Tool in your CD pipeline, you must have [required pre-requisites](docs/getting-started/pre-requisites.md) installed and configured in the CD environment.
```

Comment:
Same as other file: ...each time your check in....-> **you**

Reply:
fixed